src/components/settings/NonTandem.js

```diff
@@ -150,7 +164,6 @@ const NonTandem = (props) => {
         </div>
```

Comment:
I think we'll need a `{renderPrintNotes()}` for the non-active schedules as well.